README.md

```diff
@@ -371,6 +371,31 @@ Install a gem:
 $ gem install my_first_gem --source http://localhost:8080/gem
 ```
 
+## Helm chart repo
+
+Try this `helm.yaml` file:
+
+```yaml
+repo:
+  type: helm
+  storage:
+    type: fs
+    path: /tmp/artipie/data/helm-charts
+```
+
+Publish a chart:
+
+```bash
+$ curl --data-binary "@my_chart-1.6.4.tgz" http://localhost:8080/helm
```

Comment:
@Sammers21 helm adapter supports "helm push plugin" https://github.com/chartmuseum/helm-push see: https://github.com/artipie/helm-adapter/issues/12
So it should be `helm repo add artipie http://localhost:8080 && helm push artipie @my_chart-1.6.4.tgz`, isn't it?

Reply:
@g4s8, helm adapter is not tested with push plugin as per now and it is not as popular. I think we can add it later

Reply:
@Sammers21 ok, let's add it later